README.md

```diff
@@ -2,35 +2,13 @@
 
 This repo contains [JSON Schema](http://json-schema.org/) files and examples of the content that uses them on GOV.UK.
 
-```
-# source files and shared definitions
-formats
-└── case_study.jsonnet
-└── shared
-    └── default_format.jsonnet
-
-# built schemas (DO NOT EDIT DIRECTLY)
-dist
-└── formats
-    └── case_study
-        ├── frontend
-        │   └── schema.json
-        └── publisher
-            └── schema.json
-
-# fixtures to support testing in apps
-examples
-└── case_study
-    └── frontend
-        ├── archived.json
-        ├── case_study.json
-        └── translated.json
+The [Jsonnet](https://jsonnet.org/) source files live in `formats`, and the schemas live in `dist` (which should not be edited manually). There are also example JSON files in `examples`, to support testing in apps.
```

Comment:
```suggestion
The actual JSON schema files live in `dist` and are generated from [Jsonnet templates](https://jsonnet.org) in `source`. Using templates makes it easier to duplicate common blocks across multiple schemas e.g. format and URL fields. **Do not edit files in `dist` manually**, as they will be overwritten.
```

Optional: we can be clearer about the relationship between the two directories, and explain why we have them.